src/meta/proto-conv/tests/it/main.rs

```diff
@@ -32,3 +32,4 @@ mod v012_table_meta;
 mod v023_table_meta;
 mod v024_table_meta;
 mod v025_user_stage;
+mod v26_schema;
```

Comment:
Keep versions in alphabetic order:

```suggestion
mod v026_schema;
```